packages/dotcom-ui-header/src/__stories__/story.tsx

```diff
@@ -55,6 +55,31 @@ DefaultHeaderWithDrawer.args = {
   showLogoLink: false
 }
 
+export const EnhancedHeaderWithDrawer = (args) => (
+  <OnReady callback={onReadyCallback}>
+    <HeaderSimple {...storyData} {...args} />
+    <Drawer {...storyData} {...args} />
+    <p className="demo-sticky-message demo-sticky-message--scroll">Scroll down</p>
+  </OnReady>
+)
+
+EnhancedHeaderWithDrawer.story = {
+  name: 'Enhanced header with drawer'
+}
+EnhancedHeaderWithDrawer.args = {
+  showSubNavigation: true,
+  showMegaNav: true,
+  showUserNavigation: true,
+  userIsLoggedIn: false,
+  userIsSubscribed: false,
+  showLogoLink: false,
+  enhancedSearchProps: {
```

Comment:
Why we public `suggestionsApiUrl` and `searchUrl`? Is this safe and can we additionally protect this data?

Reply:
Why do you think it is unsafe?